portfolio/src/main/webapp/style.css

```diff
@@ -1,3 +1,30 @@
+h1{
+    font-family: courier,arial,helvetica;
+}
+
+h2{
+    font-family: courier,arial,helvetica;
+}
+
+p{
+  font-family: courier,arial,helvetica;
+  font-size: large;
+  text-align: justify;
+}
+button{
```

Comment:
nit: Add extra line between p and button so they are consistent with the rest of the file.